### PR TITLE
Writing updates2

### PR DESCRIPTION
**Pull request by jessiihu**
@katemintlify 
notes: left section about links and code page (code page contained information about inline code that wasn't  written anywhere else) 
otherwise [these updates are finished](https://www.notion.so/mintlify/Doc-updates-writing-content-f5e746708b8c4ffea74617576e6487c4)

### CODE REVIEW

File: writing-content/list-table.mdx [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+- Third item
+```
+
+<Tip>
+
+Lists follows the official [markdown syntax](https://www.markdownguide.org/basic-syntax/#lists-1).
+
+</Tip>
+
+## Tables
```
Comment (handotdev) [inline writing-content/list-table.mdx:62]: Can we add a newline after this header?

File: writing-content/image-embeds.mdx [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+<img height="200" src="/path/image.jpg" />
+```
+
+<Tip>
+
+Although not required, we recommend adding the `alt` and `title` attributes to images for better SEO and accessability. Learn more at [image SEO](https://yoast.com/image-seo-alt-tag-and-title-tag-optimization/).
+
+</Tip>
+
+### Related
```
Comment (katemintlify) [inline writing-content/image-embeds.mdx:31]: move this to the very bottom of this page

File: writing-content/image-embeds.mdx [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+<br />
+
+<Tip>
+
+Mintlify supports [HTML tags in Markdown](https://www.markdownguide.org/basic-syntax/#html). This is helpful if you prefer HTML tags to Markdown syntax, and lets you create documentation with infinite flexibility.
+
+</Tip>
+
+Here are some common HTML tags that are used for documentation
```
Comment (katemintlify) [inline writing-content/image-embeds.mdx]: is there suposed to be smth under this?

Comment (jessiihu) [reply writing-content/image-embeds.mdx]: I took it from the original embeds page, where it just had info about iframes, images, and line breaks under it. i removed the image info

Comment (katemintlify) [reply writing-content/image-embeds.mdx]: maybe remove bc it doesnt make sense

**Pull request merged**